### ISSUE TITLE
10.11 on M1, UTM 4.1 - 4.3 not working successfully

### ISSUE THREAD

**Issue by beowulff** [adespoton/utmconfigs#16]
Hi.
I was wondering if you were able to get 10.11 to run on the M1 with any success.
Way back in the UTM 3 days, I see to recall it actually worked, but in 4.1, 4.2 and now 4.3, I am not able to get it to work.
In 4.1 and 4.2, the VM actually boots, and will get to the login screen, but as I am typing my password, the emulated OS crashes and goes to a terminal screen, then, a few moments later, the login screen comes up again. So, it seems like the display manager (or some OS X system utility) is crashing.

In 4.3 (beta), I never even get to the login window. The system hangs at a terminal window with a cursor in the upper left corner, and a black background. Typing is echoed in that window, but nothing has any effect. In a few seconds, whatever I have typed is erased. The only way out is to force the VM to quit.

Have you been able to get 10.11 (or 10.12, or 10.13) to work under recent versions of UTM?

**Comment by adespoton** [comment]
Can you attach a debug log?  Looks like something has broken in 4.3, as I get stuck at the same cursor you do; every once in a while a system message pops up though.  Seems like WindowServer isn’t loading properly.  Now I need to check all the other x86 VMs.-Em On Jun 5, 2023, at 11:18 AM, beowulff ***@***.***> wrote:﻿
Hi.
I was wondering if you were able to get 10.11 to run on the M1 with any success.
Way back in the UTM 3 days, I see to recall it actually worked, but in 4.1, 4.2 and now 4.3, I am not able to get it to work.
In 4.1 and 4.2, the VM actually boots, and will get to the loving screen, but as I am typing my password, the emulated OS crashes and goes to a terminal screen, then, a few moments later, the login screen comes up again. So, it seems like the display manager (or some OS X system utility) is crashing.
In 4.3 (beta), I never even get to the login window. The system hangs at a terminal window with a cursor in the upper left corner, and a black background. Typing is echoed in that window, but nothing has any effect. In a few seconds, whatever I have typed is erased. The only way out is to force the VM to quit.
Have you been able to get 10.11 (or 10.12, or 10.13) to work under recent versions of UTM?

**Comment by adespoton** [comment]
Hmm... 10.10 may explain what's going on a bit better... it stalls at "Missing Bluetooth Controller Transport!" Which didn't happen before.

**Comment by beowulff** [comment]
Yeah, I saw that message scroll by in UTM 4.1/4.2, but it doesn’t seem to keep the system from booting.
The debug log is pretty unhelpful:
[debug.log](https://github.com/adespoton/utmconfigs/files/11657691/debug.log)

**Comment by adespoton** [comment]
I've added to your ticket in UTM; however, this appears to be an issue in the QEMU 8 code, so it'll need to go even further upstream.

**Comment by chuckaeronut** [comment]
I'll add that I can reproduce this too! I keep around a copy of UTM 3 so my old Mac VMs keep running. My El Capitan installation is my favorite, so I noticed this issue immediately on trying UTM 4.

Is there any knowledge here as to whether this is only El Capitan? Or is it all macOS x86_64 versions hitting this?

**Comment by xpitt** [comment]
I am on macbook pre m1 max 16" and now Ventura 13.5 (with the obnoxious location privacy bug). using adespoton utm settings I successfully installed os x 10.8 and 10.12 to run adobe CS5.5 and office mac 2011 that I bought years ago and don't intend to further update. After some extended testing my conclusion is that they both run best on utm 4.1.2, os x 10.8 been snappier, but also 10.12 usable. In all subsequent versions something fails, for instance safari crashes, chromium legacy crashes, firerfox legacy too. The latest utm doesn't even start 10.12 (haven't tried 10.8). Apparently something got wrong with qemu versions beyond 7.1.0 (last used in utm 4.1.2).

**Comment by xpitt** [comment]
Just tested os x 10.8 and 10.12 on utm 4.3.5. 10.8 works with some glitches: desktop does not appear initially but it does after finder force restart and safari crashes. 10.12 apparently works, so do safari and firefox (1.0.9) but both Adobe CS%.5 (acrobat, illustrator, photoshop) and Office for mac 2011 crash. So, I am going to stick as much as possible with utm 4.1.2

**Comment by adespoton** [comment]
Well that's interesting...
With 4.3.5 10.11 boots to the login screen.  When I attempt to log in, bits of the screen flicker, and then it goes to a terminal screen where it says issues were encountered accessing the TPM or something like that... then it goes back to the login screen.

So it looks like there might have been two separate problems.  I'm probably going to have to re-install from scratch and see if it behaves properly with the current virtual hardware configuration.

**Comment by xpitt** [comment]
Just tested os x 10.8 on utm 4.4.0. Same behavior as with 4.3.5

**Comment by adespoton** [comment]
If you haven’t done it already, it’s probably worth commenting on the tracking issue on the UTM repo too.-Em On Aug 27, 2023, at 4:48 AM, xpitt ***@***.***> wrote:﻿
Just tested os x 10.8 on utm 4.4.0. Same behavior as with 4.3.5

**Comment by madsmtm** [comment]
Linking it here: https://github.com/utmapp/UTM/issues/5371

**Comment by adespoton** [comment]
Looks like it's resolved now.
